openquake/sep/tests/landslide/test_displacement.py

```diff
@@ -0,0 +1,275 @@
+import unittest
+
+import numpy as np
+
+from openquake.sep.landslide.static_safety_factor import infinite_slope_fs
+
+from openquake.sep.landslide.probability import jibson_etal_2000_probability
+
+from openquake.sep.landslide.displacement import (
+    crit_accel,
+    jibson_2007_model_a,
+    jibson_2007_model_b,
+    cho_rathje_2022,
+    fotopoulou_pitilakis_2015_model_a,
+    fotopoulou_pitilakis_2015_model_b,
+    fotopoulou_pitilakis_2015_model_c,
+    fotopoulou_pitilakis_2015_model_d,
+    saygili_rathje_2008,
+    rathje_saygili_2009,
+    jibson_etal_2000,
+)
+
```

Comment:
We have conventions on the class names. You should change
```
critical_acceleration_landslide_test -> CriticalAccelerationLandslideTestCase
```
and similarly for the other classes.